Weather Alert/CoreData/Forecast+CoreDataClass.swift

```diff
@@ -10,22 +10,30 @@
 import Foundation
 import CoreData
 
-@objc(Forecast)
-public class Forecast: NSManagedObject {
-    func mapping(json: [String : Any]) {
+public class Forecast: City {
+    
+    func mapping(json: [String : Any], count: Int) {
```

Comment:
Also, it's might be good point to introduce a `protocol` if @OreniOS think so as well? I see this function `mapping` used a couple of times now. We'll speak about it tomorrow.